ios/FluentUI/Vnext/PersonaButton/PersonaButton.swift

```diff
@@ -211,22 +213,44 @@ public struct PersonaButton: View {
         self.tokens = state.tokens
     }
 
+    private var personaText: some View {
+        Group {
+            Text(state.primaryText ?? "")
+                .frame(maxWidth: .infinity, alignment: .center)
+                .scalableFont(font: tokens.labelFont)
+                .foregroundColor(Color(tokens.labelColor))
+            if state.buttonSize.shouldShowSubtitle {
+                Text(state.secondaryText ?? "")
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .scalableFont(font: tokens.sublabelFont)
+                    .foregroundColor(Color(tokens.sublabelColor))
+            }
+        }
+        .frame(width: buttonMaxWidth)
```

Comment:
shouldn't the param here be maxWidth instead of width?

Reply:
Yep, old name from a previous iteration